tutorials/en_us/PLUGINS/FAWE.md

```diff
@@ -1,5 +1,17 @@
 # How to install and use [Fast Async World Edit (FAWE)](https://www.spigotmc.org/resources/fast-async-worldedit.13932/)
 
+## WARNING
+
+FAWE can cause world corruption so use at your own risk.
+
+### How it happens
+
+FAWE asyncs things that shouldnt be async, and then flat out skips other parts of the process to speed it up. FAWE gets is speed from skipping important tasks needed for a stable experience. thats the entire reason it broke on starlight (The lighting engine tuinity uses); it skips the lighting task, and starlight only runs the lighting task once, not thousands like vanilla's. Starlight had to remove a lot of its own optimizations because people on FAWE were complaining about the plugin not working with starlight.
```

Comment:
This needs to be split into multiple lines. It has sentences that are too long and do not make sense.

### Read the sentence aloud, does it make sense while you listen to it?

Adiditonally, Tuinity, Vanilla & Starlight should be consistently capitalised.

Reply:
should make sense now

Reply:
ill keep this open just incase it doent